### ISSUE TITLE
"Select custom" > "Path" - Problems on Windows

### ISSUE THREAD

**Issue by RayCulp** [qarmin/czkawka#744]
"Select custom" > "Path" option does not seem to work on Windows, no matter how the path is entered. I have tried many combinations, such as:

```
c:\users\[username]\desktop\* (not case sensitive)

C:\users\[username]\Desktop\* (case sensitive)

c:\users\[username]\desktop* (no trailing backslash)

c:/users/[username]/desktop/* [Linux style forward slashes)

*desktop* (just to see if anything at all works)
```

**Comment by qarmin** [comment]
Which version of Czkawka?

**Comment by RayCulp** [comment]
I noticed this problem back in February of this year and started [a discussion here](https://github.com/qarmin/czkawka/discussions/639) because I wasn't sure that raising an issue immediately was the appropriate thing to do.  At that time, I was using version 4.0.0.

A few days ago, I updated to 4.0.1 and quickly tried a custom selection string, but it didn't seem to work, so I assumed this was still the same behavior.

However, after raising the issue, I continued playing around with it a bit, and I have now been able to get a few selection strings to work. 

I will do some more testing tomorrow or the next day and report back.
